__tests__/app/api/courses.jest.js

```diff
@@ -0,0 +1,197 @@
+const app = require('../../../app/app')
+const request = require('supertest')
+
+describe('GET /courses', () => {
+
+    let user
```

Comment:
could we create a global user(s) that's re-used in multiple tests?

Reply:
Agreed, there's no need to send a create account request through the API. In fact, it actually makes the test less reliable because it depends on 2 different API calls even though it's only supposed to be testing 1. A beforeAll or beforeEach (depending on which one would be best to use here - I think a beforeAll would work) should definitely be used.